lectures/inventory_ssd.md

```diff
@@ -338,9 +339,23 @@ v_init = jnp.zeros((K + 1, n_z), dtype=float)
 ```
 
 ```{code-cell} ipython3
-in_time = time.time()
```

Comment:
@shlff is there a reason you are adding `0` to the time variable name here?

Reply:
Thanks @mmcky . 

A time variable with a `0` at the end indicates that it times both compilation and execution, while a time variable without `0` considers execution time only.

Reply:
Is that distinction not clear when the text is printed? I am just wondering if the 0 is needed as `in_time` and `out_time` is clear.